qcelemental/covalent_radii.py

```diff
@@ -57,7 +58,7 @@ def __init__(self, context: str = "ALVAREZ2008"):
 
         # Extra relationships
         aliases = [
-            ('C',  'angstrom', self.cr['C_sp3'].data,       'Largest (sp3) chosen for generic atom'),
+            ('C', 'angstrom', self.cr['C_sp3'].data, 'Largest (sp3) chosen for generic atom'),
```

Comment:
it's ignoring the `yapf: disable` at the end of this list

Reply:
Yea, it doesn't do it for context. Apparently only pays attention to `yapf: disable/enable` in a non-context setting.

Reply:
uh oh, most of mine are the comment kind

Reply:
I agree, but I also removed almost all the `#yapf: disable` comments.

Reply:
ok. a lot of them in the tests could be dropped so long as it doesn't multiply number of lines by 10